.github/workflows/pr-test-generated-files.yaml

```diff
@@ -15,20 +15,23 @@ env:
 jobs:
   # Check if there is any dirty change for generated files
   generated-files:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     steps:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
           go-version: ${{ env.GO_VERSION }}
       - name: Checkout code
         uses: actions/checkout@v3
+      - name: "Install required tooling"
+        run: |
+          sudo apt install -y gcc
       - name: Check generated files
         # Note we do not use apt install -y protobuf-compiler` since it is too old
         run: |
           PB_REL="https://github.com/protocolbuffers/protobuf/releases"
           curl -LO $PB_REL/download/v${PROTO_VERSION}/protoc-${PROTO_VERSION}-linux-x86_64.zip
-          unzip protoc-${PROTO_VERSION}-linux-x86_64.zip -d $HOME/.local
+          unzip -o protoc-${PROTO_VERSION}-linux-x86_64.zip -d $HOME/.local
```

Comment:
Shouldn't we have a clean environment where we do not need to overwrite something?

Reply:
Yes that would be preferable. We could move the `.local` in the repo and add `.local` to gitignore. We need to make sure we keep a clean git repo, otherwise diff checks may fail.

Reply:
Or perhaps something like this? https://github.com/mondoohq/cnquery/pull/571/files

then we do not have to worry about the repo.